cocotb/__init__.py

```diff
@@ -137,6 +137,15 @@ def _initialise_testbench(root_name):
     """
     _rlock.acquire()
 
+    from cocotb import simulator
+
+    global SIM_NAME, SIM_VERSION
+    SIM_NAME = simulator.get_simulator_product()
+    SIM_VERSION = simulator.get_simulator_version()
+
+    cocotb.log.info("Running on {} version {}".format(SIM_NAME, SIM_VERSION))
+    cocotb.log.info("Python interpreter initialized and cocotb loaded!")
```

Comment:
This was moved to preserve the order of the log statements, but it probably shouldn't be here.